src/components/Message.svelte

```diff
@@ -7,6 +7,7 @@
   import '../css/splash.css';
   import Icon from './common/Icon.svelte';
   import Menu from './common/Menu.svelte';
+  import type { Ltl } from 'ts/typings/types.js';
```

Comment:
This isn't necessary, and isn't even imported correctly lmao.

The `Ltl` namespace is already included globally thru `tsconfig`.